dts/msc-sm2-imx6x.dtsi

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
I thought this was not working yet? If so i think we should add this to its own pull request because it is a complex and bigger dts setup.

Reply:
That's just an empty line...

Reply:
Sorry for the confusion, i meant this commit in general.